src/component/archive/ArchiveTable/ArchivedFeatureActionCell/ArchivedFeatureDeleteConfirm/ArchivedFeatureDeleteConfirm.tsx

```diff
@@ -0,0 +1,99 @@
+import { Alert, styled } from '@mui/material';
+import React, { useState } from 'react';
+import { Dialogue } from 'component/common/Dialogue/Dialogue';
+import Input from 'component/common/Input/Input';
+import { IFeatureToggle } from '../../../../../interfaces/featureToggle';
```

Comment:
We can also avoid relative paths here.

```suggestion
import { IFeatureToggle } from 'interfaces/featureToggle';
```